WordCountAlgo/WordFrequency.quorum

```diff
@@ -12,12 +12,52 @@ use Libraries.Language.Errors.UndefinedObjectError
 
 class WordFrequency
     private HashTable<text, integer> WordCountMap
+    // TODO: Populate stop words to ignore, read in file 
+    //       read in text, for each word check if it's in StopWords, if not add to map
+    //       if WordCountMap finds word in StopWords (after processing all text), RemoveKey()
+    private HashTable<text, boolean> StopWords
+
+    // constructor to initialize the list of stopwords
+    on create
+        // open stopwords file for reading
+        File stopwordsfile
+        FileReader stopwordsreader
+        stopwordsfile:SetPath("clean_stopwords-en.txt")
+        stopwordsreader:OpenForRead(stopwordsfile)
```

Comment:
instead of requiring a file on create can we use a getter and setter function to set the stopwords.